pkg/repository/udmrepo/kopialib/repo_init.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kopialib
+
+import (
+	"context"
+	"strings"
+
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/pkg/errors"
+
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo/kopialib/backend"
+)
+
+type kopiaBackendStore struct {
+	name        string
+	description string
+	store       backend.BackendStore
+}
+
+// backendStores lists the supported backend storages at present
+var backendStores []kopiaBackendStore = []kopiaBackendStore{
```

Comment:
Would it be easier to set it as a map rather than a list?

Reply:
Consider below two things, a list may be better than a map?

1. We want to bring some flexibility of the search, for example, we want to do a case insensitive compare of the `name`, because the comparing value comes from a string map of `RepoOptions`, we don't want to restrict the creator of `RepoOptions` with so many details
2. The name of `kopiaBackendStore` is an essential information of a Store, it is not only a key for the search. It means, once we have a kopiaBackendStore, we want to get its name by kopiaBackendStore.name instead of find it elsewhere, i.e., the key of a map

Reply:
OK I think it's a minor issue.  Let's keep it as a list for now.